ma/api/category.py

```diff
@@ -39,12 +39,14 @@ def delete(self, category_id):
         is_deleted = self.magento.catalog_category.delete(category_id)
         return is_deleted
 
-    def disable(self, category_id):
+    def update_active_status(self, category_id, is_active=True):
         # magento requires sorting values to be passed
-        is_disabled = self.magento.catalog_category.update(
+        is_updated = self.magento.catalog_category.update(
             category_id,
-            {'is_active': '0', 'available_sort_by': 'position', 'default_sort_by': 'position'})
-        return is_disabled
+            { 'is_active': '1' if is_active else '0',
```

Comment:
ok, will do. I thought about doing that but since Magento sometimes uses 0 for True, I wanted it to be VERY explicit and readable.